onnx/defs/tensor/defs.cc

```diff
@@ -98,10 +113,18 @@ ONNX_OPERATOR_SET_SCHEMA(
             "saturate",
             "The parameter defines how the conversion behaves if an input value is out of "
             "range of the destination type. It only applies for float 8 conversion "
-            "(float8e4m3fn, float8e4m3fnuz, float8e5m2, float8e5m2fnuz). It is true by default. "
-            "All cases are fully described in two tables inserted in the operator description.",
+            "(float8e4m3fn, float8e4m3fnuz, float8e5m2, float8e5m2fnuz, float8e8m0). It is true by default. "
+            "All cases are fully described in the tables inserted in the operator description.",
             AttributeProto::INT,
             static_cast<int64_t>(1))
+        .Attr(
+            "round_mode",
+            "Rounding mode of float8e8m0 conversion. It only applies to float8e8m0 casting and is `up` by default. "
```

Comment:
nit: I think this applies only when casting *to* float8e8m0, not when casting *from* float8e8m0. If so, can you reword to make that clear? Change `float8e8m0 conversion` to `conversion to float8e8m0` and `float8e8m0 casting` to `casting to float8e8m0`.